packages/scroller/src/styles/vaadin-scroller-base-styles.js

```diff
@@ -8,9 +8,18 @@ import { css } from 'lit';
 
 export const scrollerStyles = css`
   :host {
+    --_indicator-height: var(--vaadin-scroller-overflow-indicator-height, 1px);
+    /* Don't let these properties inherit */
+    --vaadin-scroller-padding-block: 0px;
+    --vaadin-scroller-padding-inline: 0px;
+    --vaadin-scroller-overflow-indicator-top-opacity: 0;
+    --vaadin-scroller-overflow-indicator-bottom-opacity: 0;
     display: block;
     overflow: auto;
     outline: none;
+    flex: 1;
+    box-sizing: border-box;
+    padding: 0 var(--vaadin-scroller-padding-inline);
```

Comment:
One last question: should `--vaadin-scroller-padding-block` be used here? 
I'm a bit confused as I was looking into how this should be documented.

Reply:
Got it, thanks. ~~I'll mention that these top / bottom only work with `overflow-indicators` variants then~~.
UPD: they actually work without these, I was wrong.

 Also, as variants are included to base styles, IMO we should list them in JSDoc too, same as `margin` and `padding` variants for basic layouts.